packages/front-end/components/HistoryTable.tsx

```diff
@@ -64,34 +64,60 @@ function EventDetails({
   return <Code language="json" code={JSON.stringify(json, null, 2)} />;
 }
 
-function HistoryTableRow({
+export function HistoryTableRow({
   event,
   open,
   setOpen,
+  isActivity = false,
+  itemName = "",
 }: {
   event: AuditInterface;
   open: boolean;
   setOpen: (open: boolean) => void;
+  isActivity?: boolean;
+  itemName?: string;
 }) {
+  itemName = itemName || event.entity.id;
   return (
     <>
       <tr
         onClick={() => {
-          if (event.details) setOpen(!open);
+          if (event.details && !isActivity) setOpen(!open);
         }}
         style={{ cursor: event.details ? "pointer" : "" }}
         className={open ? "highlight" : event.details ? "hover-highlight" : ""}
       >
         <td title={datetime(event.dateCreated)}>{ago(event.dateCreated)}</td>
+        {isActivity && (
+          <>
+            <td>{event.entity.object}</td>
+            <td>
+              <Link
+                href={
+                  event.entity.object === "experiment"
+                    ? `/experiment/${event.entity.id}`
+                    : `/features/${event.entity.id}`
+                }
```

Comment:
Instead of building the url here, we should pass an explicit `url` prop into the component.   If we add other item types in the future, we might forget to update this code and with how you have it now, it would link to `/features/...` with something that's not a feature id.